src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/EventDTO.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import java.time.OffsetDateTime
+
+data class EventDTO(
+  val eventType: String,
+  val description: String,
+  val detailUrl: String,
+  val occurredAt: OffsetDateTime,
+  val additionalInformation: Map<String, Any>
+) {
+  val version: Int = 1
```

Comment:
This is here to be hard coded into the DTO. It is not up to the caller to decide the version